### PR TITLE
Fix: #49 aesthetic refactor

### LINKED ISSUE #49: Aesthetic corrections, unification of the number of spaces.

### PR DESCRIPTION
**Pull request by LukasMod**

### CODE REVIEW

**Comment by LukasMod** [comment]
I updated this branch after merging branch 'About', which additionally changed the quotes to ', then reversed those changes, resolving conflicts and setting ".

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { ReactComponent as GitHub } from 'assets/github.svg';
 import './style.css';
 
 const REPO_URL = 'https://github.com/szkola-reacta/team-d-app';
-const COPYRIGHT = `Copyright © 2020 Szkoła Reacta's Team D. All Rights Reserved.`;
+const COPYRIGHT = `Copyright © 2s020 Szkoła Reacta's Team D. All Rights Reserved.`;
```
Comment (weronikaolejniczak) [inline src/components/Footer/Footer.js]: What happened here?

**Review by weronikaolejniczak — APPROVED** [review]

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (WronskiN) [reply src/components/Footer/Footer.js]: Sssssnake 👯

**Review by WronskiN — APPROVED** [review]

File: src/components/Footer/Footer.js (hunk 1 above)
Comment (LukasMod) [reply src/components/Footer/Footer.js]: sssssnake fixed. no idea

**Pull request merged**